templates/macvtap.yaml.in

```diff
@@ -24,7 +24,21 @@ spec:
         volumeMounts:
           - name: deviceplugin
             mountPath: /var/lib/kubelet/device-plugins
+        command: [ "./macvtap", "-v", "3", "-logtostderr"]
```

Comment:
Full path would be probably better. Or export PATH in the Dockerfile and use just `macvtap`

Reply:
Done.